src/dotty/tools/dotc/repl/CompilingInterpreter.scala

```diff
@@ -6,7 +6,7 @@ import java.io.{
   File, PrintWriter, PrintStream, StringWriter, Writer, OutputStream,
   ByteArrayOutputStream => ByteOutputStream
 }
-import java.lang.{Class, ClassLoader}
+import java.lang.{Class, ClassLoader, Thread, System, StringBuffer}
```

Comment:
See commit message "Honor the new scheme where any explicit import of a root import will disable the root import."